perftools/stresstesting/login-and-learn.yml

```diff
@@ -0,0 +1,22 @@
+config:
```

Comment:
Having a new top-level folder called "perftools", could that be made better somehow? What's the intention of this in the context of this PR and in an overall context?